File: PRAK605-2310817210001-AndraBraputraAkbarSaleh.py
def multiply(matrix_a, matrix_b):
    result_matrix = []
    
    for i in range(len(matrix_a)):
        row = []
        for j in range(len(matrix_b[0])):
            element = 0
            for k in range(len(matrix_a[0])):
                element += matrix_a[i][k] * matrix_b[k][j]
            row.append(element)
        result_matrix.append(row)
    
    return result_matrix

def main():
    n = int(input())
    print ("Matrix A")
    matrix_a = []
    for _ in range(n):
        row = list(map(int, input().split()))
        matrix_a.append(row)
    
    print ("Matrix B")
    matrix_b = []
    for _ in range(n):
        row = list(map(int, input().split()))
        matrix_b.append(row)
    
    result_matrix = multiply(matrix_a, matrix_b)
    
    
    print ("Matrix AXB")
    for row in result_matrix:
        print(' '.join(map(str, row)))

if __name__ == "__main__":
    main()

File: PRAK605-2310817210001-AndraBraputraAkbarSaleh.c
#include <stdio.h>

void multiply(int n, int matrix_a[n][n], int matrix_b[n][n], int result_matrix[n][n]) {
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            result_matrix[i][j] = 0;
            for (int k = 0; k < n; ++k) {
                result_matrix[i][j] += matrix_a[i][k] * matrix_b[k][j];
            }
        }
    }
}

int main() {
    int n;
    scanf("%d", &n);

    int matrix_a[n][n];
    printf("Matrix A\n");
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            scanf("%d", &matrix_a[i][j]);
        }
    }

    int matrix_b[n][n];
    printf("Matrix B\n");
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            scanf("%d", &matrix_b[i][j]);
        }
    }

    int result_matrix[n][n];
    multiply(n, matrix_a, matrix_b, result_matrix);

    printf("Matrix AXB:\n");
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < n; ++j) {
            printf("%d ", result_matrix[i][j]);
        }
        printf("\n");
    }

    return 0;
}
